package.json

```diff
@@ -84,10 +84,10 @@
     "lodash": "^4.0.0",
     "mocha": "^3.0.0",
     "mocha-dynamic-tests": "^2.0.0",
-    "node-sass": "^3.4.2",
-    "react": "^16.12.0",
+    "node-sass": "^6.0.1",
```

Comment:
@tmuzanenhamo  are we sticking with text-mask?
Also you can drop node-sass i think